### PR TITLE
Added Org & Repository variables

### PR DESCRIPTION
**Pull request by Froxz**

### CODE REVIEW

File: lib/Github/Api/Organization/Actions/Variables.php [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            throw new MissingArgumentException(['name']);
+        }
+
+        if (!isset($parameters['value'])) {
+            throw new MissingArgumentException(['value']);
+        }
+
+        if (!isset($parameters['visibility'])) {
+            throw new MissingArgumentException(['visibility']);
+        }
```
Comment (acrobat) [inline lib/Github/Api/Organization/Actions/Variables.php:60]: I would not add this validation in our code, but let the github api validate the request and return any errors related to missing parameters. This way we won't block user in the future if github changes the required parameters

File: lib/Github/Api/Repository/Actions/Variables.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+     */
+    public function create(string $username, string $repository, array $parameters = [])
+    {
+        if (!isset($parameters['name'])) {
+            throw new MissingArgumentException(['name']);
+        }
+
+        if (!isset($parameters['value'])) {
+            throw new MissingArgumentException(['value']);
+        }
```
Comment (acrobat) [inline lib/Github/Api/Repository/Actions/Variables.php:57]: I would not add this validation in our code, but let the github api validate the request and return any errors related to missing parameters. This way we won't block user in the future if github changes the required parameters

**Review by acrobat — CHANGES_REQUESTED** [review]

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thank you @Froxz

**Comment by Froxz** [comment]
@acrobat thank you

**Pull request merged**